.github/workflows/release.yml

```diff
@@ -194,149 +177,6 @@ jobs:
           tag_name: ${{ matrix.config.tagName }}
           files: ./sbom-${{ matrix.config.name }}.spdx.json
 
-  release-manifests:
```

Comment:
manifest releases are removed for now and readded later with `helm template` from the KLT umbrella chart

Reply:
should we put any particular TODO here?

Reply:
I already added the task to the DoD of the umbrella chart ticket. I don't think we need to add it here as well.